[2023-02-13T17:53:44.700+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_pg_opv3.create_table scheduled__2023-02-12T15:23:00+00:00 [queued]>
[2023-02-13T17:53:44.712+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_pg_opv3.create_table scheduled__2023-02-12T15:23:00+00:00 [queued]>
[2023-02-13T17:53:44.712+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-13T17:53:44.712+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-02-13T17:53:44.712+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-13T17:53:44.736+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): create_table> on 2023-02-12 15:23:00+00:00
[2023-02-13T17:53:44.751+0000] {standard_task_runner.py:55} INFO - Started process 63 to run task
[2023-02-13T17:53:44.757+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_pg_opv3', 'create_table', 'scheduled__2023-02-12T15:23:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_pg_op.py', '--cfg-path', '/tmp/tmppz09qt79']
[2023-02-13T17:53:44.757+0000] {standard_task_runner.py:83} INFO - Job 112: Subtask create_table
[2023-02-13T17:53:44.916+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_pg_opv3.create_table scheduled__2023-02-12T15:23:00+00:00 [running]> on host 29b44968227e
[2023-02-13T17:53:45.038+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=naufal
AIRFLOW_CTX_DAG_ID=dag_pg_opv3
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-02-12T15:23:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-02-12T15:23:00+00:00
[2023-02-13T17:53:45.038+0000] {sql.py:254} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt,dag_id)
            );
        
[2023-02-13T17:53:45.050+0000] {base.py:73} INFO - Using connection ID 'pg_docker' for task execution.
[2023-02-13T17:53:45.312+0000] {base.py:73} INFO - Using connection ID 'pg_docker' for task execution.
[2023-02-13T17:53:45.321+0000] {sql.py:375} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt,dag_id)
            );
        , parameters: None
[2023-02-13T17:53:45.333+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=dag_pg_opv3, task_id=create_table, execution_date=20230212T152300, start_date=20230213T175344, end_date=20230213T175345
[2023-02-13T17:53:45.380+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-13T17:53:45.401+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
